### ISSUE TITLE
No /tmp directory on windows systems

### ISSUE THREAD

**Issue by responseinformationsdesign** [kartolo/direct_mail#95]
As there is no /tmp directory on windows systems debug output results in an error.

`Core: Error handler (BE): PHP Warning: file_put_contents(/tmp/dmail.php): failed to open stream: No such file or directory in C:\wwwroot\...\typo3conf\ext\direct_mail\Classes\DirectMailUtility.php line 1227`

**Comment by fgebauer** [comment]
Same problem here but on a linux host.

The code in lines 1226-1227 of `Classes\DirectMailUtility.php`  looks strange like a forgotten debug code.
```php
 // !ian save last dmail source in tmp for debug
 file_put_contents('/tmp/dmail.php', var_export($htmlmail->theParts, true));
```
It breaks also on other systems, if the php user cannot write in the /tmp directory for security reasons or others.

**Comment by kartolo** [comment]
fixed in #96
